hadoop-hdds/docs/content/feature/ContainerBalancerOverview.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Container Balancer overview"
+weight: 10
+menu:
+   main:
+      parent: Features
+summary: Container Balancer overview.
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+## Container Balancer overview
```

Comment:
Yes, my original comment about feature overview was referring to this. Based on this the user gets an idea about what the container balancer is and what it does. Thank you for adding it.